sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -229,7 +229,7 @@ def check_regression_dataset(loss, subsample):
 
         y_pred = reg.predict(X_reg)
         mse = mean_squared_error(y_reg, y_pred)
-        assert mse < 0.04
+        assert mse < 0.05
```

Comment:
Is this PR introducing a change in the results of some estimators?

Reply:
I put this on the side for the moment and I was going to check more in depth.

It appears on 32 bits only thought: https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=19128&view=logs&j=240d9ac2-2494-53e1-bb8d-1d2c3e4c80b2&t=050bd822-9ae7-5d24-4fa8-ba7d2d004fa5&l=333

Reply:
I thought that the interpolation between adjacent ranks with higher strategy was equivalent to the nearest-rank search but apparently not exactly. So I added an option `interpolation=None` to keep the previous behaviour which does not use the interpolation approach and directly the nearest-rank.